packages/ms-request-api/src/IssuerUtil.ts

```diff
@@ -0,0 +1,30 @@
+import { IIssueRequest, IIssueRequestResponse } from "./types/IMsRequestApi";
+
+export async function fetchIssuanceRequestMs(issuanceInfo: IIssueRequest, accessToken: string, msIdentityHostName: string): Promise<IIssueRequestResponse> {
+    var client_api_request_endpoint = `${msIdentityHostName}${issuanceInfo.authenticationInfo.azTenantId}/verifiablecredentials/request`;
+
+    var payload = JSON.stringify(issuanceInfo.issuanceConfig);
+    const fetchOptions = {
+        method: 'POST',
+        body: payload,
+        headers: {
+            'Content-Type': 'application/json',
+            'Content-Length': payload.length.toString(),
+            'Authorization': `Bearer ${accessToken}`
+        }
+    };
+    const response = await fetch(client_api_request_endpoint, fetchOptions);
+    return await response.json();
+}
+
+export function generatePin(digits: any) {
```

Comment:
Is there a reason why digits are "any"?

Reply:
generatePin parameter changed any->number